sys-firmware/libkrunfw/libkrunfw-4.0.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit toolchain-funcs python-any-r1
+
+MY_KVER="6.4.7"
+
+DESCRIPTION="Linux kernel shared object for libkrun"
+HOMEPAGE="https://github.com/containers/libkrunfw"
+SRC_URI="
+	https://github.com/containers/libkrunfw/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-${MY_KVER}.tar.xz
+"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~arm64"
+IUSE="asahi"
+
+DEPEND="
+	${PYTHON_DEPS}
+"
+BDEPEND="
+	dev-python/pyelftools
```

Comment:
You still need `python_gen_any_dep`, per leio's comments and https://projects.gentoo.org/python/guide/any.html#dependencies.